hub/api/dataset.py

```diff
@@ -255,24 +261,46 @@ def mode(self):
     def pytorch(
         self,
         transform: Optional[Callable] = None,
-        workers: int = 1,
+        num_workers: int = 1,
         tensors: Optional[List[str]] = None,
+        batch_size: Optional[int] = 1,
+        drop_last: Optional[bool] = False,
+        collate_fn: Optional[Callable] = None,
+        pin_memory: Optional[bool] = False,
     ):
-        """Converts the dataset into a pytorch compatible format.
+        """Converts the dataset into a pytorch Dataloader.
 
         Note:
             Pytorch does not support uint16, uint32, uint64 dtypes. These are implicitly type casted to int32, int64 and int64 respectively.
-            This spins up it's own workers to fetch data, when using with torch.utils.data.DataLoader, set num_workers = 0 to avoid issues.
+            This spins up it's own workers to fetch data.
 
         Args:
-            transform (Callable, optional) : Transformation function to be applied to each sample
-            workers (int): The number of workers to use for fetching data in parallel.
-            tensors (List, optional): Optionally provide a list of tensor names in the ordering that your training script expects. For example, if you have a dataset that has "image" and "label" tensors, if `tensors=["image", "label"]`, your training script should expect each batch will be provided as a tuple of (image, label).
+            transform (Callable, optional) : Transformation function to be applied to each sample.
+            num_workers (int): The number of workers to use for fetching data in parallel.
```

Comment:
_Additional_ workers? If so, why default to 1 and not 0?

Reply:
yep that's what i was saying

Reply:
if we default to 0 though i think it doesn't work with multiprocessing and abhinav said he was going to have it default to old pytorch, because it's non-trivial to add 0 workers for multiprocessing. but we opted to just keep default to 0 and implement that later to reduce scope